regexes/client/browsers.yml

```
@@ -5,6 +5,11 @@
 # @license http://www.gnu.org/licenses/lgpl.html LGPL v3 or later
 ###############
 
+# Light (https://sourceforge.net/projects/lightfirefox/)
+- regex: 'Light/(?:(\d+[\.\d]+))?'
```

If we require the version to be present we can simplify that to
```suggestion
- regex: 'Light/(\d+[\.\d]+)'
```